hail_scripts/elasticsearch/elasticsearch_utils.py

```diff
@@ -100,11 +100,17 @@ def elasticsearch_schema_for_table(table, disable_doc_values_for_fields=(), disa
         logger.info("==> will disable index fields for %s", ", ".join(disable_index_for_fields))
         for es_field_name in disable_index_for_fields:
             if es_field_name not in properties:
-                raise ValueError(
-                    "'%s' in disable_index_for_fields arg is not in the elasticsearch schema: %s"
-                    % (es_field_name, properties)
-                )
-            properties[es_field_name]["index"] = False
+                flattened_fields = [key for key in properties if f"{es_field_name}_" in key]
```

Comment:
So the key has to start with the prefix not just contain it, so it should not be `f"{es_field_name}_" in key` it should be `key.startswith(f'{es_field_name}_')